### ISSUE TITLE
Security patch causing issues

### ISSUE THREAD

**Issue by tdcox** [dokterdok/Continuity-Activation-Tool#285]
Looks like the latest security patches from Apple are causing conflict with CAT, at least on an Early 2011 MBP with updated radio. Ethernet and Wi-Fi interfaces are broken.

**Comment by ezdub** [comment]
Agreed. I can't remember agreeing to security patch install but after a reboot yesterday, the wi-fi tray icon began reporting "no wi-fi hardware installed" until I disabled sip via recovery mode in the process of reverting CAT.   Mid 2011MBAir on v10.11.3 (15D21)

**Comment by th0u** [comment]
I can second that MBAir 2011 latest os. "no wi-fi hardware installed" 
I had to uninstall all continuity patches
